docs/pages/CLI.md

```diff
@@ -1,253 +1,325 @@
-  
-  ui5 versions
-      Shows the versions of all UI5 Tooling modules
-
-Options:
-  --help, -h               Show help  [boolean]
-  --version, -v            Show version number  [boolean]
-  --config                 Path to configuration file  [string]
-  --translator, --t8r      Translator to use. Including optional colon separated translator parameters.  [string] [default: "npm"]
-  --verbose                Enable verbose logging.[boolean]
-  --loglevel, --log-level  Set the logging level (error|warn|info|verbose|silly).  [string] [default: "info"]
-
-Examples:
+### Usage/Syntax
+
+`
+ ui5 <command> [options]
+`
+
+### Common options
+
+These options you can use with each command.
+
+| Option | Description |
+| --- | --- |
+|   --help, -h              | Show help  [boolean] |
+|   --version, -v           | Show version number  [boolean] |
+|   --config                | Path to configuration file  [string] |
+|   --translator, --t8r     | Translator to use. Including optional colon separated translator parameters.  [string] [default: &quot;npm&quot;] |
+|   --verbose               | Enable verbose logging.  [boolean] |
+|   --loglevel, --log-level | Set the logging level (error\|warn\|info\|verbose\|silly).  [string] [default: &quot;info&quot;] |
+
+### Examples
+
+Execute command using a &quot;static&quot; translator with translator parameters
+```
   ui5 <command> --translator static:/path/to/projectDependencies.yaml
-            Execute command using a "static" translator with translator
-              parameters
-  ui5 <command> --config /path/to/ui5.yaml
-            Execute command using a project configuration from custom path
 ```
-
-The CLI automatically checks for updates using [update-notifier](https://github.com/yeoman/update-notifier). While this is skipped in CI environments, you might also opt-out manually by following the steps described [here](https://github.com/yeoman/update-notifier/blob/master/readme.md#user-settings).
+Execute command using a project configuration from custom path
+```
+  ui5 <command> --config /path/to/ui5.yaml                           
+```
 
 ### Commands
-#### add
 
-`ui5 add [--development] [--optional] <framework-libraries..>` adds SAPUI5/OpenUI5 framework libraries to the project configuration
+#### ui5 add
+
+**Description**
 
+Add SAPUI5/OpenUI5 framework libraries to the project configuration.
+
+**Usage**
+
+`
+ui5 add [--development] [--optional] <framework-libraries..>
+`
+
+
+**Options**
+
+| Option | Description |
+| --- | --- |
+|   --development, -D, --dev | Add as development dependency  [boolean] [default: false] |
+|   --optional, -O           | Add as optional dependency  [boolean] [default: false] |
+
+**Positionals**
+
+| Positional | Description |
+| --- | --- |
+|   framework-libraries | Framework library names  [array] [required] [default: []] |
+
+**Examples**
+
+Add the framework libraries sap.ui.core and sap.m as dependencies
+```
+  ui5 add sap.ui.core sap.m              
```

Comment:
It would be good to trim the whitespaces here. In many bash configurations, commands preceded by a whitespace are not added to the bash history, which might not be desirable.